src/graphql/schema/withOperators.ts

```diff
@@ -9,122 +9,211 @@ import operators from './operators';
 
 type staticTypes = 'number' | 'text' | 'email' | 'textarea' | 'richText' | 'json' | 'code' | 'checkbox' | 'date' | 'upload' | 'point' | 'relationship'
 
+type dynamicTypes = 'radio' | 'select'
+
+const GeoJSONObject = new GraphQLInputObjectType({
+  name: 'GeoJSONObject',
+  fields: {
+    type: { type: GraphQLString },
+    coordinates: {
+      type: GraphQLJSON,
+    },
+  },
+});
+
 type DefaultsType = {
   [key in staticTypes]: {
-    type: GraphQLType | ((field: FieldAffectingData, parentName: string) => GraphQLType);
-    operators: string[];
+    operators: {
+      operator: string;
```

Comment:
If we named this `name` or `value` or something else we could improve readability later on where we have: `operator.operator` which is hard to understand.